src/tools/tests/schema2kotlin-test.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+
+import {assert} from '../../platform/chai-node.js';
+import {KotlinGenerator} from '../schema2kotlin.js';
+import {SchemaNode} from '../schema2graph.js';
+import {Schema} from '../../runtime/schema.js';
+
+
+describe('schema2wasm', () => {
+  describe('kotlin-generator', () => {
+    it('when no items are present, it creates an empty map', () => {
+      const ktGen = new KotlinGenerator(new SchemaNode(new Schema([], {}), 'dummyNode'), {arg: '', _: []});
```

Comment:
You could put this either in a `beforeEach` or at the top of the file. No need to reconstruct it every time.